perf/benchmark/runner/runner.py

```diff
@@ -129,6 +131,13 @@ def serversidecar(self, fortio_cmd):
         return fortio_cmd + "_serveronly " + basestr.format(
             svc=self.server.ip, port=self.ports[self.mode]["port"], size=self.size)
 
+    def clientsidecar(self, fortio_cmd):
+        basestr = "http://{svc}:{port}/echo?size={size}"
+        if self.mode == "grpc":
+            basestr = "-payload-size {size} {svc}:{port}"
+        return fortio_cmd + "_clientonly " + basestr.format(
+            svc=self.client.ip, port=self.ports[self.mode]["port"], size=self.size)
```

Comment:
I don't quite understand here, this doesn't look correct, you are sending from client to client?